internal/backends/winit/event_loop.rs

```diff
@@ -535,6 +563,13 @@ pub fn run() {
                 *control_flow = ControlFlow::Poll;
             }
 
+            for window in windows_with_pending_redraw_requests
+                .drain(..)
+                .filter_map(|window_id| window_by_id(window_id))
+            {
+                window.draw();
```

Comment:
Can we draw from here?
Maybe we can try to call request_redraw()  again instead?

Reply:
Yes, we can draw from here.